src/helpers/validateSignUpEmailAddress.js

```diff
@@ -0,0 +1,13 @@
+import { ALPHANUMERIC } from "./validateEmailAddress";
+
+const validateSignUpEmailAddress = (email) => {
+    const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN
+
+    const EMAIL_DOMAIN_REGEX_STRING = emailDomain.replace(',','|').toUpperCase();
+    const EMAIL_VALIDATION_STRING =
+    `^[${ALPHANUMERIC}.!#$%&'*+/=?^_\`{|}~-]` + 
+    `+@${EMAIL_DOMAIN_REGEX_STRING}`
```

Comment:
Add `([A-Za-z0-9\-]\.)*` here, to capture subdomains. `test@trust.nhs.uk` is a possible email address.